coalib/results/Result.py

```diff
@@ -60,8 +60,8 @@ def __init__(self,
             A message which may help the user find out why this result was
             yielded.
         :param diffs:
-            A dictionary with filenames as key and a sequence of ``Diff``
-            objects associated with them as values.
+            A dictionary with filename as key and `Diff` object
```

Comment:
what about double backticks? I think we should use those, I've read somewhere that those do link to the actual classes or so in RST. Not really sure, but it's anyway more consistent then^^